crates/re_renderer/src/context.rs

```diff
@@ -106,6 +123,32 @@ impl RenderContext {
     ) -> Self {
         re_tracing::profile_function!();
 
+        let frame_index_for_uncaptured_errors = Arc::new(AtomicU64::new(STARTUP_FRAME_IDX));
+
+        // Make sure to catch all errors, never crash and deduplicate reported errors.
+        // `on_uncaptured_error` is a last-resort handler which we should never hit,
+        // since there should always be an open error scope.
+        //
+        // Note that this handler may not be called for all errors if no scope is present.
```

Comment:
I'm having trouble parsing this. "If there is no open scope, then this handler MAY miss some errors." maybe?